runbooks/upp-content-es-runbook.md

```diff
@@ -0,0 +1,96 @@
+# UPP - Content Elasticsearch Cluster
+
+Elasticsearch cluster for use by the Content Search API Port service and Content-RW-Elasticsearch service. 
+
+## Code
+
+upp-content-es-cluster
+
+## Service Tier
+
+Platinum
+
+## Lifecycle Stage
+
+Production
+
+## Delivered By
+
+content
+
+## Supported By
+
+content
+
+## Known About By
+
+- hristo.georgiev
+- robert.marinov
+- elina.kaneva
+- georgi.ivanov
+- tsvetan.dimitrov
+- dimitar.terziev
```

Comment:
Add more people here. E.g. I always add Platform health team. We aim to have ~10 people in known about.

Reply:
Ok, I did not know that we aim for ~10 people per system, I will add PH team as well.